api/src/extensions/manager.ts

```diff
@@ -173,7 +175,8 @@ export class ExtensionManager {
 	 */
 	private async load(): Promise<void> {
 		try {
-			await ensureExtensionDirs(env['EXTENSIONS_PATH'], NESTED_EXTENSION_TYPES);
+			await syncExtensions();
+			await ensureExtensionDirs(getExtensionsPath(), NESTED_EXTENSION_TYPES);
```

Comment:
Can we shift `ensureExtensionDirs()` into `syncExtensions()` so that only the primary process creates the required directories during the sync?

Reply:
We can!

Reply:
Updated. Thanks! 👍